### ISSUE TITLE
prometheus_extra_opts: [] Does not appear to be functioning correctly.

### ISSUE THREAD

**Issue by tfalkowski-nsidc** [mesaguy/ansible-prometheus#19]
Hey mesaguy,

Once again thanks for your work on this module it is truly appreciated.

Think I have found a bug. It seems that anything being defined in the prometheus_extra_opts: [] array is summarily ignored during deploy. There is no error the options are just not set in the prometheus.service file when the build finishes. For reference,  I am trying to pass the following storage options:
```
prometheus_extra_opts:
  -  '--storage.tsdb.wal-compression'
  -  '--storage.tsdb.retention.time=31d'
  -  '--storage.tsdb.retention.size=20GB'
```
I am curious if something changed with the last release? I feel like your module used to default to having the the storage.tsdb.retention flag set to a default of 90days if no other options were defined in the vars but that does not seem to be the case anymore.

Thank you in advance for taking a look at this.

**Comment by mesaguy** [comment]
Thank you for the feedback and identifying this problem. I have a solution that I am actively testing and the fix will be released in the next couple of days. Sorry for the trouble.

I am fairly confident that this role has only had 90d storage retention as a README.md example and the default retention period has been Prometheus' default.

**Comment by mesaguy** [comment]
This issue has been solved by 851bc89852deff09e68f013981e276fb477c1a50. v0.12.0 has been released with this fix.
